test/resources/omop-dicomweb/extract_summary.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
its a shame we have to duplicate this, but I think we probably do have to so that we have another endpoint for the config. Could we just slim down the json files to what we actually use?

Reply:
We could look into some sort of "smart" json solution such as jsonnet so we don't just have to copy and paste everything.

Reply:
>its a shame we have to duplicate this, but I think we probably do have to so that we have another endpoint for the config. Could we just slim down the json files to what we actually use?
I agree but this would require some knowledge of what all these fields actually mean and what they're used for... which I currently don't have 😅

Reply:
we basically only care about the project name and the datetime. Look through the code for what we use and strip out everything else?